mig/shared/defaults.py

```diff
@@ -76,6 +77,33 @@
 any_protocol = keyword_any
 any_state = keyword_any
 
+from pwd import getpwuid
+mig_user = {
+    'default': 'mig',
+    'local': getpwuid(os.getuid())[0],
+}[MIG_ENV]
+
+from grp import getgrgid
+mig_group = {
+    'default': 'mig',
+    'local': getgrgid(os.getgid())[0],
+}[MIG_ENV]
+
+default_source = {
+    'default': keyword_auto,
+    'local': os.path.join(MIG_BASE, "mig/install"),
+}[MIG_ENV]
+
+default_destination = {
+    'default': keyword_auto,
+    'local': os.path.join(MIG_BASE, "envhelp/output/confs"),
+}[MIG_ENV]
+
+default_enable_events = {
```

Comment:
Changed _upstream_ default to False now.

Reply:
Cool. It doesn't really affect the PR, except I guess this could be removed. I think I'd prefer we just land this if it works rather than messing with it even more, but what would you prefer?

Reply:
Yeah, sorry for the noise. I had a number of simplifications in mind with more `keywords_auto` defaults to eliminate the need to mangle inside shared.defaults at all and this was really just to prepare for that. Let's discuss and sync when to do what internally.